tvb/simulator/plot/utils/data_structures_utils.py

```diff
@@ -0,0 +1,691 @@
+# -*- coding: utf-8 -*-
```

Comment:
All modules under utils should be reviewed, and not copied here unless really used in plotting. I suspect many are not imported.

Reply:
Ok. I created a utils.py with all the used methods from plot utils.